ibis/expr/api.py

```diff
@@ -4658,6 +4658,7 @@ def _rowid(self):
     'anti_join': _regular_join_method('anti_join', 'anti'),
     'asof_join': asof_join,
     'sort_by': _table_sort_by,
+    'order_by': _table_sort_by,
```

Comment:
-0.5 on this. duplicating names causes even more friction. no objection to adding some working to the doc-string though.

Reply:
Maybe it would be preferable to rename `sort_by` to `order_by`, and deprecate `sort_by`?  Equivalent in the near-term and still cleans up the API in the longer term.

Reply:
> Maybe it would be preferable to rename sort_by to order_by, and deprecate sort_by? Equivalent in the near-term and still cleans up the API in the longer term.

-1 on that. this is a long established api.

Reply:
I don't think there's a huge amount to be gained by duplicating (opens the Pandora's box of questions about why there are two) or by renaming (breaks a lot of existing code).

I would however like to see a dead simple mapping of SQL constructs to ibis apis in a quickstart. This would decrease friction without disruption. I'm all for disruption but I don't think this particular breaking change gives us much bang for our buck.

Reply:
SQL predates the Ibis API by 30+ years.  `order_by` is intuitive to data folks and obeys the [Principle of Least Surprise](https://en.wikipedia.org/wiki/Principle_of_least_astonishment).  We're trying to make API easier for larger numbers of people to use, and adding an alias like this is one small thing we can do.

Reply:
That's fair. One option is to remove references to sort_by and deprecate it in 4.0. We've got a good chunk of breaking API changes slated for 3.0. How about a deprecation in 3.0, full removal in 4.0? I agree that order_by is what folks coming from SQL will immediately reach for.

Reply:
Just noting, that there are users coming from pandas as well where the term `sort` is used.

Reply:
Definitely quite a debate...whatever solution is selected to move forward, updating docs can never hurt, been scouring docs the past few weeks so will flag stuff when I see it....on another note would be nice to have examples of direct comparisons of SQL --> ibis in the docs with functions like `.diff` as well as `merge_asof`,  Maybe biased coming from a mostly time series perspective...